src/graphql/error-map.ts

```diff
@@ -481,6 +482,8 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
     case "LikelyNoUserWithThisPhoneExistError":
     case "LikelyUserAlreadyExistError":
     case "PhoneIdentityDoesNotExistError":
+    case "UserWithPhoneAlreadyExistsError":
```

Comment:
We could probably add a friendly message for this error since it is very specific and relevant to the calling user

Reply:
Fixed here - 6df07700713571070501cc450952a51b96e5b031